Makefile

```diff
@@ -181,7 +181,7 @@ ensure-yarn: # Yarn ensures the correct version of yarn is installed
 
 yarn:
 	corepack enable
-	corepack prepare yarn@stable --activate
+	corepack prepare yarn@$(YARN_VERSION) --activate
```

Comment:
Is this change intended? It doesn't seem applicable to this PR

Reply:
`make yarn` fails with `Usage Error: Invalid package manager specification in CLI arguments; expected a semver version` otherwise. I figured I would include it as a minor dev fix, but if you think it would be better as its own PR then I can revert it.

Reply:
Sounds good. I ran into this too at some point. I updated the description.